tests/pass/shims/fs.rs

```diff
@@ -209,7 +208,10 @@ fn test_file_set_len() {
 
     // Can't use set_len on a file not opened for writing
     let file = OpenOptions::new().read(true).open(&path).unwrap();
-    assert_eq!(ErrorKind::InvalidInput, file.set_len(14).unwrap_err().kind());
+    assert_eq!(
+        if cfg!(windows) { ErrorKind::PermissionDenied } else { ErrorKind::InvalidInput },
```

Comment:
Since we pass through the host OS error, I doubt this will work properly. Just check that it's one of the two (with a comment explaining why).